package.json

```diff
@@ -1,42 +1,139 @@
-  "main": "dist/lib/index.js",
-  "repository": "github.com/DataDog/datadog-cdk-constructs",
-  "author": "Datadog",
-  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:DataDog/datadog-cdk-constructs.git"
+  },
   "scripts": {
-    "build": "tsc",
-    "watch": "tsc -w",
-    "test": "jest",
-    "lint": "tslint --project tsconfig.json",
-    "format": "prettier --write lib/**",
-    "check-formatting": "prettier --check lib/**",
-    "cdk": "cdk"
+    "projen": "npx projen",
+    "start": "npx projen start",
+    "clobber": "npx projen clobber",
+    "check-formatting": "npx projen check-formatting",
+    "compile": "npx projen compile",
+    "test:compile": "npx projen test:compile",
+    "test": "npx projen test",
+    "build": "npx projen build",
+    "bump": "npx projen bump",
+    "release": "npx projen release",
+    "test:watch": "npx projen test:watch",
+    "test:update": "npx projen test:update",
+    "projen:upgrade": "npx projen projen:upgrade",
+    "watch": "npx projen watch",
+    "package": "npx projen package",
+    "eslint": "npx projen eslint",
+    "compat": "npx projen compat"
+  },
+  "author": {
+    "name": "Datadog",
+    "organization": true
   },
   "devDependencies": {
     "@aws-cdk/assert": "^1.71.0",
     "@aws-cdk/aws-lambda": "^1.71.0",
     "@aws-cdk/aws-logs": "^1.71.0",
     "@aws-cdk/aws-logs-destinations": "^1.71.0",
     "@aws-cdk/core": "^1.71.0",
-    "@types/jest": "^26.0.19",
-    "@types/node": "^10.17.27",
-    "@typescript-eslint/eslint-plugin": "^4.9.1",
-    "@typescript-eslint/parser": "^4.9.1",
+    "@types/jest": "^26.0.20",
+    "@types/node": "^10.17.0",
+    "@typescript-eslint/eslint-plugin": "^4.16.1",
+    "@typescript-eslint/parser": "^4.16.1",
     "aws-cdk": "^1.71.0",
+    "eslint": "^7.21.0",
+    "eslint-config-prettier": "^8.1.0",
+    "eslint-import-resolver-node": "^0.3.4",
+    "eslint-import-resolver-typescript": "^2.4.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-prettier": "^3.3.1",
     "jest": "^26.6.3",
-    "prettier": "2.2.1",
-    "ts-jest": "^26.2.0",
-    "ts-node": "^8.1.0",
-    "tslint": "^6.1.3",
-    "typescript": "^3.9.7"
+    "jest-junit": "^12",
+    "jsii": "^1.24.0",
+    "jsii-diff": "^1.24.0",
+    "jsii-pacmak": "^1.24.0",
+    "json-schema": "^0.3.0",
+    "prettier": "^2.2.1",
+    "projen": "^0.16.36",
+    "standard-version": "^9",
+    "ts-jest": "^26.5.3",
+    "ts-node": "^9.1.1",
+    "typescript": "^4.2.3"
   },
   "peerDependencies": {
     "@aws-cdk/aws-lambda": "^1.71.0",
     "@aws-cdk/aws-logs": "^1.71.0",
     "@aws-cdk/aws-logs-destinations": "^1.71.0",
-    "@aws-cdk/core": "^1.71.0"
-  }
-}
+    "@aws-cdk/core": "^1.71.0",
+    "constructs": "^3.2.27"
+  },
+  "bundledDependencies": [],
+  "keywords": [
+    "cdk"
+  ],
+  "main": "lib/index.js",
+  "license": "Apache-2.0",
+  "version": "0.0.0",
```

Comment:
I realize this should reflect the latest version of the DD CDK, I am investigating how version bumping works with projen.

Reply:
Yeah we definitely want to have the right version number here.